src/config/index.ts

```diff
@@ -0,0 +1,11 @@
+import { extendTheme } from "@chakra-ui/react";
+
+import { checkboxTheme } from "./checkbox";
+
+const theme = extendTheme({
+  components: {
+    Checkbox: checkboxTheme,
+  },
+});
+
+export default theme;
```

Comment:
would prefer not to have this if we dont need to

Reply:
These are custom styles for checkbox. When I was polishing the design I tried to go with the green color from Chakra as primary/brand color. I am not able to style the checkbox nicely in the file where it is so this is global styling for all checkboxes so we have them with the same design.

How do you like keeping the green as primary?

Reply:
alright fine with me ig, assuming this doesnt degrade performance or add to bundle size?